api/api/models/__init__.py

```diff
@@ -105,6 +114,23 @@ def send_confirm_email(self) -> int:
         url = f'https://{os.environ["DOMAIN_NAME"]}/confirm?token={tmp_token}'
         cache.set(tmp_token, self.email, CONFIRM_TOKEN_TIMEOUT_SECONDS)
 
+        # Define the URL
```

Comment:
why do this here ?

Reply:
you should write a standalone function for it ; this function should be called from the view you have already added. this prevents users to confirm their accounts